index.js

```diff
@@ -1595,30 +1596,37 @@ export const createJavaBom = async (path, options) => {
           );
           options.failOnError && process.exit(1);
         } else {
+          const target = process.env.BAZEL_TARGET || "//...";
+          let query;
+          let bazelParser;
+          if (process.env.BAZEL_USE_ACTION_GRAPH === "true") {
```

Comment:
We generally accept both true and 1

https://github.com/CycloneDX/cdxgen/blob/master/utils.js#L88